packages/postcss-colormin/src/__tests__/index.js

```diff
@@ -6,7 +6,7 @@ import {
     processCSSWithPresetFactory,
 } from '../../../../util/testHelpers';
 
-const {passthroughCSS, processCSS} = processCSSFactory(plugin);
+const { passthroughCSS, processCSS } = processCSSFactory(plugin);
```

Comment:
This is causing issues with our linting settings.

Reply:
@andyjansson Sorry, I've missed that and fixed it with the next commit.